lib/helpers/tasks.rb

```diff
@@ -47,14 +47,9 @@ def run_prefix_command(payload)
   end
 
   def run_command(command)
-    if Open3.respond_to?('capture3')
-      stdout, stderr, exit_status = Open3.capture3(command)
-      message = "triggered: #{command}\n#{stdout}\n#{stderr}"
-    else
-      message = "forked: #{command}"
-      Process.detach(fork { exec "#{command} &" })
-      exit_status = 0
-    end
+    message = "forked: #{command}"
+    exec "#{command} &"
+    exit_status = 0
     raise "#{stdout}\n#{stderr}" if exit_status != 0
```

Comment:
Isn't this line redundant if you explicitly set `exit_status = 0`?

Reply:
@ekohl Would it be better to remove that or replace `exit_status = 0` with `exit_status = exec "echo $?`

Reply:
I'd remove it because it's forked and the output is useless:
```
ekohl@wisse ~ $ false &
[1] 13302
[1]+  Exit 1                  false
ekohl@wisse ~ $ echo $?
0
ekohl@wisse ~ $ /usr/bin/sfdsdfsfs &
[1] 13324
bash: /usr/bin/sfdsdfsfs: No such file or directory
[1]+  Exit 127                /usr/bin/sfdsdfsfs
ekohl@wisse ~ $ echo $?
0
```